com.hazelcast.jet.hadoop.file.ParquetFileFormatTest.shouldThrowWhenIncorrectSchema[0: useHadoop=true]

_master_ (commit c194f25f0dd1e58c1f4712d0a71598992bda6896)

Failed on Zulu JDK 11 and IBM JDK 8: 
- http://jenkins.hazelcast.com/job/jet-oss-master-zulu-jdk11/305/testReport/junit/com.hazelcast.jet.hadoop.file/ParquetFileFormatTest/shouldThrowWhenIncorrectSchema_0__useHadoop_true_/
- http://jenkins.hazelcast.com/job/jet-oss-master-ibm-jdk8/298/testReport/junit/com.hazelcast.jet.hadoop.file/ParquetFileFormatTest/shouldThrowWhenIncorrectSchema_0__useHadoop_true_/

Stacktrace for Zulu JDK 11:
```
java.lang.AssertionError: 

Expecting throwable message:
  <"com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')">
to contain:
  <"com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to com.hazelcast.jet.hadoop.file.model.IncorrectUser">
but did not.

Throwable that failed the check:

java.util.concurrent.CompletionException: com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at java.base/java.util.concurrent.CompletableFuture.reportJoin(CompletableFuture.java:412)
	at java.base/java.util.concurrent.CompletableFuture.join(CompletableFuture.java:2044)
	at com.hazelcast.jet.Job.join(Job.java:148)
	at com.hazelcast.jet.hadoop.file.ParquetFileFormatTest.lambda$shouldThrowWhenIncorrectSchema$0(ParquetFileFormatTest.java:109)
	at org.assertj.core.api.ThrowableAssert.catchThrowable(ThrowableAssert.java:62)
	at org.assertj.core.api.AssertionsForClassTypes.catchThrowable(AssertionsForClassTypes.java:750)
	at org.assertj.core.api.Assertions.catchThrowable(Assertions.java:1179)
	at org.assertj.core.api.Assertions.assertThatThrownBy(Assertions.java:1051)
	at com.hazelcast.jet.hadoop.file.ParquetFileFormatTest.shouldThrowWhenIncorrectSchema(ParquetFileFormatTest.java:109)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
	at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
	at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:62)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:373)
	at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807)
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:346)
	... 1 more
Caused by: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47)
	at com.hazelcast.jet.core.processor.Processors.lambda$mapP$5abdc455$1(Processors.java:698)
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47)
	at com.hazelcast.jet.core.AbstractProcessor$FlatMapper.tryProcess(AbstractProcessor.java:567)
	at com.hazelcast.jet.impl.processor.TransformP.tryProcess(TransformP.java:45)
	at com.hazelcast.jet.core.AbstractProcessor.tryProcess0(AbstractProcessor.java:187)
	at com.hazelcast.jet.core.AbstractProcessor.process0(AbstractProcessor.java:591)
	at com.hazelcast.jet.core.AbstractProcessor.process(AbstractProcessor.java:109)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.processInbox(ProcessorTasklet.java:416)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:291)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:286)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:268)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:242)
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:366)
	... 3 more

	at com.hazelcast.jet.hadoop.file.ParquetFileFormatTest.shouldThrowWhenIncorrectSchema(ParquetFileFormatTest.java:112)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
	at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
	at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:62)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Stacktrace for IBM JDK 8:
```
java.lang.AssertionError: 

Expecting throwable message:
  <"com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: com.hazelcast.jet.hadoop.file.generated.SpecificUser incompatible with com.hazelcast.jet.hadoop.file.model.IncorrectUser">
to contain:
  <"com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to com.hazelcast.jet.hadoop.file.model.IncorrectUser">
but did not.

Throwable that failed the check:

java.util.concurrent.CompletionException: com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: com.hazelcast.jet.hadoop.file.generated.SpecificUser incompatible with com.hazelcast.jet.hadoop.file.model.IncorrectUser
	at java.util.concurrent.CompletableFuture.reportJoin(CompletableFuture.java:386)
	at java.util.concurrent.CompletableFuture.join(CompletableFuture.java:1958)
	at com.hazelcast.jet.Job.join(Job.java:148)
	at com.hazelcast.jet.hadoop.file.ParquetFileFormatTest.lambda$shouldThrowWhenIncorrectSchema$0(ParquetFileFormatTest.java:109)
	at com.hazelcast.jet.hadoop.file.ParquetFileFormatTest$$Lambda$1760/00000000F5E1AE70.call(Unknown Source)
	at org.assertj.core.api.ThrowableAssert.catchThrowable(ThrowableAssert.java:62)
	at org.assertj.core.api.AssertionsForClassTypes.catchThrowable(AssertionsForClassTypes.java:750)
	at org.assertj.core.api.Assertions.catchThrowable(Assertions.java:1179)
	at org.assertj.core.api.Assertions.assertThatThrownBy(Assertions.java:1051)
	at com.hazelcast.jet.hadoop.file.ParquetFileFormatTest.shouldThrowWhenIncorrectSchema(ParquetFileFormatTest.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
	at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
	at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:62)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.util.concurrent.FutureTask.run(FutureTask.java:277)
	at java.lang.Thread.run(Thread.java:822)
Caused by: com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: com.hazelcast.jet.hadoop.file.generated.SpecificUser incompatible with com.hazelcast.jet.hadoop.file.model.IncorrectUser
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:373)
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker$$Lambda$1210/00000000F47901D0.accept(Unknown Source)
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:906)
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:346)
	... 1 more
Caused by: java.lang.ClassCastException: com.hazelcast.jet.hadoop.file.generated.SpecificUser incompatible with com.hazelcast.jet.hadoop.file.model.IncorrectUser
	at com.hazelcast.jet.hadoop.file.ParquetFileFormatTest$$Lambda$1761/00000000E40047A0.applyEx(Unknown Source)
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47)
	at com.hazelcast.jet.core.processor.Processors.lambda$null$1565dc65$1(Processors.java:698)
	at com.hazelcast.jet.core.processor.Processors$$Lambda$1766/00000000E402A840.applyEx(Unknown Source)
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47)
	at com.hazelcast.jet.core.AbstractProcessor$FlatMapper.tryProcess(AbstractProcessor.java:567)
	at com.hazelcast.jet.impl.processor.TransformP.tryProcess(TransformP.java:45)
	at com.hazelcast.jet.core.AbstractProcessor.tryProcess0(AbstractProcessor.java:187)
	at com.hazelcast.jet.core.AbstractProcessor.process0(AbstractProcessor.java:591)
	at com.hazelcast.jet.core.AbstractProcessor.process(AbstractProcessor.java:109)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.processInbox(ProcessorTasklet.java:416)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:291)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:286)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:268)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:242)
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:366)
	... 4 more

	at com.hazelcast.jet.hadoop.file.ParquetFileFormatTest.shouldThrowWhenIncorrectSchema(ParquetFileFormatTest.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
	at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
	at com.hazelcast.test.metrics.MetricsRule$1.evaluate(MetricsRule.java:62)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.util.concurrent.FutureTask.run(FutureTask.java:277)
	at java.lang.Thread.run(Thread.java:822)

```

Standard output for Zulu JDK 11:
```
2020-12-10 04:23:38,267 [ INFO] [Time-limited test] [o.a.h.i.c.CodecPool]: Got brand-new compressor [.snappy]
2020-12-10 04:23:38,652 [ INFO] [Time-limited test] [o.a.p.h.InternalParquetRecordWriter]: Flushing mem columnStore to file. allocated memory: 2048
2020-12-10 04:23:38,801 [ INFO] [Time-limited test] [c.h.i.m.i.MetricsConfigHelper]: [LOCAL] [jet] [4.4-SNAPSHOT] Overridden metrics configuration with system property 'hazelcast.metrics.collection.frequency'='1' -> 'MetricsConfig.collectionFrequencySeconds'='1'
2020-12-10 04:23:38,802 [ INFO] [Time-limited test] [c.h.system]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Hazelcast Jet 4.4-SNAPSHOT (20201210 - c194f25) starting at [127.0.0.1]:5701
2020-12-10 04:23:38,803 [ INFO] [Time-limited test] [c.h.system]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Based on Hazelcast IMDG version: 4.1.1 (20201209 - a64039a)
2020-12-10 04:23:38,803 [ INFO] [Time-limited test] [c.h.system]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Cluster name: jet
2020-12-10 04:23:38,803 [ INFO] [Time-limited test] [c.h.system]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] 
	o   o   o   o---o o---o o     o---o   o   o---o o-o-o        o o---o o-o-o
	|   |  / \     /  |     |     |      / \  |       |          | |       |
	o---o o---o   o   o-o   |     o     o---o o---o   |          | o-o     |
	|   | |   |  /    |     |     |     |   |     |   |      \   | |       |
	o   o o   o o---o o---o o---o o---o o   o o---o   o       o--o o---o   o
2020-12-10 04:23:38,803 [ INFO] [Time-limited test] [c.h.system]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
2020-12-10 04:23:38,807 [ INFO] [Time-limited test] [c.h.i.m.i.MetricsConfigHelper]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Collecting debug metrics and sending to diagnostics is enabled
2020-12-10 04:23:38,817 [ WARN] [Time-limited test] [c.h.c.CPSubsystem]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] CP Subsystem is not enabled. CP data structures will operate in UNSAFE mode! Please note that UNSAFE mode will not provide strong consistency guarantees.
2020-12-10 04:23:38,832 [ INFO] [Time-limited test] [c.h.j.i.e.TaskletExecutionService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Creating idler with jet.idle.cooperative.min.microseconds=25?s,jet.idle.cooperative.max.microseconds=500?s
2020-12-10 04:23:38,832 [ INFO] [Time-limited test] [c.h.j.i.e.TaskletExecutionService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Creating idler with jet.idle.noncooperative.min.microseconds=25?s,jet.idle.noncooperative.max.microseconds=5000?s
2020-12-10 04:23:38,845 [ INFO] [Time-limited test] [c.h.j.i.JetService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Setting number of cooperative threads and default parallelism to 72
2020-12-10 04:23:38,930 [ INFO] [Time-limited test] [c.h.i.d.Diagnostics]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Diagnostics disabled. To enable add -Dhazelcast.diagnostics.enabled=true to the JVM arguments.
2020-12-10 04:23:38,930 [ INFO] [Time-limited test] [c.h.c.LifecycleService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] [127.0.0.1]:5701 is STARTING
2020-12-10 04:23:38,937 [ INFO] [Time-limited test] [c.h.i.c.ClusterService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] 

Members {size:1, ver:1} [
	Member [127.0.0.1]:5701 - 819d81e9-e315-4189-bd58-a819bb9e0dc8 this
]

2020-12-10 04:23:38,938 [DEBUG] [hz.nifty_torvalds.cached.thread-1] [c.h.j.i.JobCoordinationService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Not starting jobs because partitions are not yet initialized.
2020-12-10 04:23:38,950 [ INFO] [Time-limited test] [c.h.c.LifecycleService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] [127.0.0.1]:5701 is STARTED
2020-12-10 04:23:38,980 [ INFO] [hz.nifty_torvalds.cached.thread-3] [c.h.i.p.i.PartitionStateManager]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Initializing cluster partition table arrangement...
2020-12-10 04:23:39,020 [DEBUG] [hz.nifty_torvalds.cached.thread-3] [c.h.j.i.p.Planner]: Watermarks in the pipeline will be throttled to 1000
2020-12-10 04:23:39,055 [ INFO] [hz.nifty_torvalds.cached.thread-3] [c.h.j.i.JobCoordinationService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Starting job 0566-f7d6-31c0-0001 based on submit request
2020-12-10 04:23:39,056 [DEBUG] [hz.nifty_torvalds.cached.thread-1] [c.h.j.i.JobRepository]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Job cleanup took 14ms
2020-12-10 04:23:39,118 [ INFO] [hz.nifty_torvalds.cached.thread-3] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Didn't find any snapshot to restore for job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001
2020-12-10 04:23:39,118 [ INFO] [hz.nifty_torvalds.cached.thread-3] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Start executing job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001, execution graph in DOT format:
digraph DAG {
	"readHadoop" [localParallelism=2];
	"map" [localParallelism=72];
	"loggerSink" [localParallelism=1];
	"readHadoop" -> "map" [queueSize=1024];
	"map" -> "loggerSink" [queueSize=1024];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
2020-12-10 04:23:39,118 [DEBUG] [hz.nifty_torvalds.cached.thread-3] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Building execution plan for job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001
2020-12-10 04:23:39,123 [ INFO] [hz.nifty_torvalds.cached.thread-3] [o.a.h.m.l.i.FileInputFormat]: Total input files to process : 1
2020-12-10 04:23:39,123 [ INFO] [hz.nifty_torvalds.cached.thread-3] [o.a.p.h.ParquetInputFormat]: Total input paths to process : 1
2020-12-10 04:23:39,136 [ INFO] [hz.nifty_torvalds.cached.thread-3] [c.h.j.h.i.R.readHadoop#ProcessorMetaSupplier]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Split counts per member before uniquifying: [1]
2020-12-10 04:23:39,137 [ INFO] [hz.nifty_torvalds.cached.thread-3] [c.h.j.h.i.R.readHadoop#ProcessorMetaSupplier]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Final split counts per member: [1]
2020-12-10 04:23:39,137 [ INFO] [hz.nifty_torvalds.cached.thread-3] [c.h.j.h.i.R.readHadoop#ProcessorMetaSupplier]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Member-to-split assignment: [127.0.0.1]:5701:
IndexedInputSplit{index 0, blocks  [localhost]}, locations [localhost]}
2020-12-10 04:23:39,147 [DEBUG] [hz.nifty_torvalds.cached.thread-3] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Built execution plans for job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001
2020-12-10 04:23:39,160 [DEBUG] [hz.nifty_torvalds.cached.thread-3] [c.h.j.i.o.InitExecutionOperation]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Initializing execution plan for job 0566-f7d6-31c0-0001, execution 0566-f7d6-31c1-0001 from [127.0.0.1]:5701
2020-12-10 04:23:39,264 [ INFO] [hz.nifty_torvalds.cached.thread-3] [c.h.j.i.JobExecutionService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Execution plan for jobId=0566-f7d6-31c0-0001, jobName='0566-f7d6-31c0-0001', executionId=0566-f7d6-31c1-0001 initialized
2020-12-10 04:23:39,265 [DEBUG] [hz.nifty_torvalds.cached.thread-1] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Init of job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001 was successful
2020-12-10 04:23:39,265 [DEBUG] [hz.nifty_torvalds.cached.thread-1] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Executing job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001
2020-12-10 04:23:39,265 [ INFO] [hz.nifty_torvalds.cached.thread-1] [c.h.j.i.JobExecutionService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Start execution of job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001 from coordinator [127.0.0.1]:5701
2020-12-10 04:23:39,322 [ INFO] [hz.nifty_torvalds.jet.blocking.thread-0] [o.a.p.h.InternalParquetRecordReader]: RecordReader initialized will read a total of 2 records.
2020-12-10 04:23:39,322 [ INFO] [hz.nifty_torvalds.jet.blocking.thread-0] [o.a.p.h.InternalParquetRecordReader]: at row 0. reading next block
2020-12-10 04:23:39,329 [ INFO] [hz.nifty_torvalds.jet.blocking.thread-0] [o.a.h.i.c.CodecPool]: Got brand-new decompressor [.snappy]
2020-12-10 04:23:39,331 [ INFO] [hz.nifty_torvalds.jet.blocking.thread-0] [o.a.p.h.InternalParquetRecordReader]: block read in memory in 9 ms. row count = 2
2020-12-10 04:23:39,371 [ WARN] [hz.nifty_torvalds.jet.cooperative.thread-3] [c.h.j.i.e.TaskletExecutionService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Exception in ProcessorTasklet{map#1}
java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.processor.Processors.lambda$mapP$5abdc455$1(Processors.java:698) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor$FlatMapper.tryProcess(AbstractProcessor.java:567) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.processor.TransformP.tryProcess(TransformP.java:45) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.tryProcess0(AbstractProcessor.java:187) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.process0(AbstractProcessor.java:591) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.process(AbstractProcessor.java:109) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.processInbox(ProcessorTasklet.java:416) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:291) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:286) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:268) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:242) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:366) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807) [?:?]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:346) [hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:834) [?:?]
2020-12-10 04:23:39,371 [ WARN] [hz.nifty_torvalds.jet.cooperative.thread-2] [c.h.j.i.e.TaskletExecutionService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Exception in ProcessorTasklet{map#0}
java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.processor.Processors.lambda$mapP$5abdc455$1(Processors.java:698) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor$FlatMapper.tryProcess(AbstractProcessor.java:567) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.processor.TransformP.tryProcess(TransformP.java:45) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.tryProcess0(AbstractProcessor.java:187) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.process0(AbstractProcessor.java:591) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.process(AbstractProcessor.java:109) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.processInbox(ProcessorTasklet.java:416) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:291) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:286) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:268) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:242) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:366) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807) [?:?]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:346) [hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:834) [?:?]
2020-12-10 04:23:39,374 [DEBUG] [hz.nifty_torvalds.jet.cooperative.thread-45] [c.h.j.i.JobExecutionService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Execution of job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001 completed with failure
java.util.concurrent.CompletionException: com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331) ~[?:?]
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:632) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506) ~[?:?]
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088) ~[?:?]
	at com.hazelcast.jet.impl.util.NonCompletableFuture.internalCompleteExceptionally(NonCompletableFuture.java:59) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$ExecutionTracker.taskletDone(TaskletExecutionService.java:461) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.dismissTasklet(TaskletExecutionService.java:392) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:378) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807) [?:?]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:346) [hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:373) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	... 3 more
Caused by: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.processor.Processors.lambda$mapP$5abdc455$1(Processors.java:698) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor$FlatMapper.tryProcess(AbstractProcessor.java:567) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.processor.TransformP.tryProcess(TransformP.java:45) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.tryProcess0(AbstractProcessor.java:187) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.process0(AbstractProcessor.java:591) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.process(AbstractProcessor.java:109) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.processInbox(ProcessorTasklet.java:416) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:291) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:286) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:268) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:242) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:366) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	... 3 more
2020-12-10 04:23:39,378 [DEBUG] [ForkJoinPool.commonPool-worker-59] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Execution of job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001 has failures: [[127.0.0.1]:5701=com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')]
2020-12-10 04:23:39,378 [DEBUG] [hz.nifty_torvalds.cached.thread-14] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Sending CompleteExecutionOperation for job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001
2020-12-10 04:23:39,379 [DEBUG] [hz.nifty_torvalds.cached.thread-14] [c.h.j.i.o.CompleteExecutionOperation]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Completing execution 0566-f7d6-31c1-0001 from caller [127.0.0.1]:5701, error=com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
2020-12-10 04:23:39,379 [DEBUG] [hz.nifty_torvalds.cached.thread-14] [c.h.j.i.JobExecutionService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Completed execution of job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001
2020-12-10 04:23:39,379 [ERROR] [hz.nifty_torvalds.cached.thread-14] [c.h.j.i.MasterJobContext]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Execution of job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001 failed
	Start time: 2020-12-10T04:23:39.055
	Duration: 00:00:00.324
	To see additional job metrics enable JobConfig.storeMetricsAfterJobCompletion
com.hazelcast.jet.JetException: Exception in ProcessorTasklet{map#1}: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:373) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807) ~[?:?]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:346) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:834) ~[?:?]
Caused by: java.lang.ClassCastException: class com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to class com.hazelcast.jet.hadoop.file.model.IncorrectUser (com.hazelcast.jet.hadoop.file.generated.SpecificUser and com.hazelcast.jet.hadoop.file.model.IncorrectUser are in unnamed module of loader 'app')
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.processor.Processors.lambda$mapP$5abdc455$1(Processors.java:698) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.function.FunctionEx.apply(FunctionEx.java:47) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor$FlatMapper.tryProcess(AbstractProcessor.java:567) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.processor.TransformP.tryProcess(TransformP.java:45) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.tryProcess0(AbstractProcessor.java:187) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.process0(AbstractProcessor.java:591) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.core.AbstractProcessor.process(AbstractProcessor.java:109) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.processInbox(ProcessorTasklet.java:416) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:291) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:286) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:268) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:242) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:366) ~[hazelcast-jet-4.4-SNAPSHOT.jar:4.4-SNAPSHOT]
	... 3 more
2020-12-10 04:23:39,381 [DEBUG] [hz.nifty_torvalds.cached.thread-14] [c.h.j.i.JobCoordinationService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] job '0566-f7d6-31c0-0001', execution 0566-f7d6-31c1-0001 is completed
2020-12-10 04:23:39,385 [DEBUG] [Time-limited test] [c.h.j.i.JobCoordinationService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Added a shutting-down member: 819d81e9-e315-4189-bd58-a819bb9e0dc8
2020-12-10 04:23:39,386 [ INFO] [Time-limited test] [c.h.c.LifecycleService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] [127.0.0.1]:5701 is SHUTTING_DOWN
2020-12-10 04:23:39,386 [ INFO] [hz.nifty_torvalds.cached.thread-14] [c.h.i.p.i.MigrationManager]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Shutdown request of Member [127.0.0.1]:5701 - 819d81e9-e315-4189-bd58-a819bb9e0dc8 this is handled
2020-12-10 04:23:39,387 [ INFO] [Time-limited test] [c.h.i.i.Node]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Shutting down connection manager...
2020-12-10 04:23:39,387 [ INFO] [Time-limited test] [c.h.i.i.Node]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Shutting down node engine...
2020-12-10 04:23:39,427 [ INFO] [Time-limited test] [c.h.i.i.NodeExtension]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Destroying node NodeExtension.
2020-12-10 04:23:39,429 [ INFO] [Time-limited test] [c.h.i.i.Node]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Hazelcast Shutdown is completed in 43 ms.
2020-12-10 04:23:39,430 [ INFO] [Time-limited test] [c.h.c.LifecycleService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] [127.0.0.1]:5701 is SHUTDOWN
2020-12-10 04:23:39,430 [ INFO] [Time-limited test] [c.h.j.c.JetTestSupport]: Terminating instanceFactory in JetTestSupport.@After
2020-12-10 04:23:39,430 [ INFO] [Thread-36] [c.h.c.LifecycleService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] [127.0.0.1]:5701 is SHUTTING_DOWN
2020-12-10 04:23:39,430 [ INFO] [Thread-36] [c.h.i.i.Node]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] Node is already shutting down... Waiting for shutdown process to complete...
2020-12-10 04:23:39,431 [ INFO] [Thread-36] [c.h.c.LifecycleService]: [127.0.0.1]:5701 [jet] [4.4-SNAPSHOT] [127.0.0.1]:5701 is SHUTDOWN
BuildInfo right after shouldThrowWhenIncorrectSchema[0: useHadoop=true](com.hazelcast.jet.hadoop.file.ParquetFileFormatTest): BuildInfo{version='4.1.1-SNAPSHOT', build='20201209', buildNumber=20201209, revision=a64039a, enterprise=false, serializationVersion=1, jet=JetBuildInfo{version='4.4-SNAPSHOT', build='20201210', revision='c194f25'}}
Hiccups measured while running test 'shouldThrowWhenIncorrectSchema[0: useHadoop=true](com.hazelcast.jet.hadoop.file.ParquetFileFormatTest):'
04:23:35, accumulated pauses: 252 ms, max pause: 58 ms, pauses over 1000 ms: 0
```

--- a/extensions/hadoop/src/test/java/com/hazelcast/jet/hadoop/file/ParquetFileFormatTest.java
+++ b/extensions/hadoop/src/test/java/com/hazelcast/jet/hadoop/file/ParquetFileFormatTest.java
@@ -109,8 +109,8 @@ public class ParquetFileFormatTest extends BaseFileFormatTest {
             assertThatThrownBy(() -> jets[0].newJob(p).join())
                     .hasCauseInstanceOf(JetException.class)
                     .hasRootCauseInstanceOf(ClassCastException.class)
-                    .hasMessageContaining("com.hazelcast.jet.hadoop.file.generated.SpecificUser cannot be cast to "
-                            + "com.hazelcast.jet.hadoop.file.model.IncorrectUser");
+                    .hasMessageContaining("com.hazelcast.jet.hadoop.file.generated.SpecificUser")
+                    .hasMessageContaining("com.hazelcast.jet.hadoop.file.model.IncorrectUser");
         } finally {
             for (JetInstance jet : jets) {
                 jet.shutdown();

